pkg/api/types.go

```diff
@@ -1672,7 +1672,7 @@ type NodeSpec struct {
 // DaemonEndpoint contains information about a single Daemon endpoint.
 type DaemonEndpoint struct {
 	// Port number of the given endpoint.
-	Port int `json:port`
+	Port int `json:"port"`
```

Comment:
@goltermann @krousey 

Yes.

So long as pkg/api/types.go still contains json tags, they should not differ from the versioned API without good reason.

Please suggest a clarification (via PR) to:
https://github.com/kubernetes/kubernetes/blob/master/docs/devel/api_changes.md#edit-typesgo-1